### DISCUSSION TITLE
S1 support

### DISCUSSION THREAD

**Discussion by grrmisfit** [Jyers/Marlin#1869] [Q&A]
Is there a beta for S! support? I know it was said by tito that his next release would support it, but Ihadnt seen anything on it. currrently using that other guys and the bugs are driving me mad and stock is as bad. plus i cant fix the thermal failure bug since you cant compile his code

**Comment by thedroidkid** [comment]


**Reply by grrmisfit** [reply]
one thing I would add about the S1 is the marlin false positives on thermal runaways, this happened on my S1 and was a nightmare till I saw the issue on marlins github and their fix solved my issues. https://github.com/MarlinFirmware/Marlin/issues/23581

**Reply by dut97** [reply]
@tititopher68-dev 
I'm highly interested in the S1 firmware. Any idea on the ETA?

**Reply by yozza1412** [reply]
im checking this everyday praying that Jyers firmware will be available for the S1..

**Reply by yozza1412** [reply]
any eta guys ?
